models/tasks.js

```diff
@@ -86,8 +91,11 @@ const fetchActiveTaskMembers = async () => {
 const fetchTask = async (taskId) => {
   try {
     const task = await tasksModel.doc(taskId).get()
-    const taskData = task.data()
-    return { taskData: await fromFirestoreData(taskData) }
+    const taskData = await fromFirestoreData(task.data())
+    if (taskData?.status) {
+      taskData.status = TASK_STATUS[taskData.status.toUpperCase()]
```

Comment:
**Enhancement**: Let's have the default case as **IN_PROGRESS** everywhere because superusers can modify task status from the browser, while doing that if he/she makes a typo then task status will be undefined right?

Reply:
having `IN_PROGRESS` doesn't feel right here? Since it's a typo, then we should treat it as a typo and should have a different name for it, by which anyone can identify that status is wrong here.